.github/workflows/test-node.js.yml

```diff
@@ -28,4 +28,6 @@ jobs:
       uses: actions/setup-node@v2
       with:
         node-version: ${{ matrix.node-version }}
-    - run: npm cit
+    - run: npm ci --no-audit
```

Comment:
minor question: Can you quickly describe the motivation behind adding `--no-audit` here?

Reply:
Yes:
a) (Since we have tools to provide update PRs) nobody is looking at the audit output on CI
b) [it saves time to not do it](http://www.tiernok.com/posts/2019/faster-npm-installs-during-ci/)